changelogs/.plugin-cache.yaml

```diff
@@ -0,0 +1,278 @@
+plugins:
```

Comment:
Please add this to `.gitignore`and remove this file from the PR

Reply:
I will push that up today. Thank you!